setupTests.js

```diff
@@ -0,0 +1 @@
+import '@testing-library/jest-dom/extend-expect';
```

Comment:
same.

I think we can use `src/testutil/setupTest.ts` (.ts file) for this one.